docs/addons/addon-migration-guide.md

```diff
@@ -1,77 +1,67 @@
 
-Or use a version specifier:
+### Node.js 16 Support Dropped
 
-```json
-{
-  "dependencies": {
-    "@storybook/client-logger": "^7.0.0"
-  }
-}
-```
+Please upgrade your addon to Node.js 18, as support for Node.js 16 has ended.
 
-## Breaking changes
+### React 18 for Manager UI
 
-### `@storybook/addons` has been split into `@storybook/manager-api` and `@storybook/preview-api`
+UI injected into panels, tools, etc. by addons is now rendered with React 18. Also note that the `key` prop is no longer passed to the render function.
 
-The default export from `@storybook/addons` can now be used via named imports from `@storybook/manager-api` and `@storybook/preview-api`, depending on which environment you need the API from. The manager is the Storybook UI and includes your addon's Addon Panel. While the preview is used to render stories and includes your addon's decorators.
+### @storybook/components Deprecations
 
-You might also depend (and use) these packages in your addon's decorators: `@storybook/store`, `@storybook/preview-web`, `@storybook/core-client`, `@storybook/client-api`. These have all been consolidated into `@storybook/preview-api`. If you use any of these packages, please import what you need from `@storybook/preview-api` instead.
+`Icons` component from `@storybook/components` is now deprecated in favor of [`@storybook/icons`](https://github.com/storybookjs/icons). Additionally, various `Button` component props are also deprecated, with alternatives provided.
 
-### Some components were moved from `@storybook/components` to a new package `@storybook/blocks`
+### Storybook Layout State API Changes
 
-Components like `ColorControl`, `ColorPalette`, `ArgsTable`, `ArgRow`, `TabbedArgsTable`, `SectionRow`, `Source`, AND `Code` were moved into a new package. In Storybook 7.0, they should be imported from `@storybook/blocks` instead.
+If you're customizing the Storybook UI configuration with `addons.setConfig({...})`, be aware of [the changes to the layout state API](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#ui-layout-state-has-changed-shape).
 
-```js
-// import { ColorControl } from '@storybook/components';
-import { ColorControl } from '@storybook/blocks';
-```
+### Removal of Deprecated Features
 
-## Deprecations and detailed migrations
+Deprecated packages and APIs from 7.0 are now removed in 8.0.Consult the [full migration notes](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecations-which-are-now-removed) for details. Most notably for addons, the removal of the `@storybook/addons` now necessitates a switch to `@storybook/preview-api` and `@storybook/manager-api`.
 
-We’ve also deprecated a few packages and APIs in 7.0. After you’ve made your addon working with 7.0, make sure to check the browser console in a Storybook running your addon. If you’re using deprecated packages, you should see warnings that link to migration instructions.
+### Babel-loader removed from Webpack
 
```

Comment:
@kylegach mind reviewing this bit that just got added? thanks!

Reply:
@vanessayuenn can you address the comments that I left in #25660 as there are a couple of items to address? THanks

Reply:
oh gosh sorry I missed those!